### PR TITLE
[docs] First pass on improving the GCP installation guide

### PR DESCRIPTION
**Pull request by jankeromnes**

### CODE REVIEW

**Comment by jankeromnes** [comment]
https://deploy-preview-620--gitpod-website.netlify.app/docs/self-hosted/0.4.0/install/install-on-gcp-script/

File: src/docs/self-hosted/0.4.0/install/install-on-gcp-script.md [hunk 1]
```diff
@@ -4,43 +4,33 @@ url: /docs/self-hosted/0.4.0/install/install-on-gcp-script/
...
+Alternatively, you can set up the GCP resources manually yourself. See the [manual GCP setup](../install-on-gcp-manual/) section for more details.
 
 ## Prerequisites
-- A Google Cloud project
-- [Domain](../domain)
-- [Google Cloud SDK](https://cloud.google.com/sdk/install)
-- [Go (at least 1.12)](https://golang.org/doc/install)
-- MySQL client
+
+- A [Google Cloud](https://cloud.google.com/) project
```
Comment (geropl) [inline src/docs/self-hosted/0.4.0/install/install-on-gcp-script.md:22]: Add link to the [free trial](https://console.cloud.google.com/freetrial) and mention the 300$ budget for new users.

File: src/docs/self-hosted/0.4.0/install/install-on-gcp-script.md [hunk 2]
```diff
@@ -4,43 +4,33 @@ url: /docs/self-hosted/0.4.0/install/install-on-gcp-script/
...
+- A [Domain](../domain)
 - Optional: [HTTPS Certificates](../https-certs)
 
 ## Configuration
 
-The [Gitpod self-hosted repository](https://github.com/gitpod-io/self-hosted) contains the configuration files this guide is refering to.
-The installation script will modify the configuration files found in that repository.
-We recommend you to fork this repository so that you can easily rebase your changes on the latest version.
+The [Gitpod self-hosted repository](https://github.com/gitpod-io/self-hosted) contains all the base configuration files.
```
Comment (geropl) [inline src/docs/self-hosted/0.4.0/install/install-on-gcp-script.md:28]: -`base`

File: src/docs/self-hosted/0.4.0/install/install-on-gcp-script.md [hunk 3]
```diff
@@ -4,43 +4,33 @@ url: /docs/self-hosted/0.4.0/install/install-on-gcp-script/
 
 # Install Gitpod on Google Cloud Platform
 
-At its core setting up Gitpod on GCP does not differ much from installing it on any other Kubernetes platform.
-However, Gitpod sports a range of integrations with the Google Cloud Platform that increase performance and reliability.
-This section describes the neccesary steps to prepare a new GCP project for a Gitpod installation.
+Gitpod runs best on GCP, because that's also where [gitpod.io](https://gitpod.io) is deployed and operated at scale.
 
-  > We have automated this process. If you want to get up and running as quickly as possible, that's the way to go.
+The easiest way to install Gitpod on GCP is by using our automated installation script. You can find it in this repository:
```
Comment (geropl) [inline src/docs/self-hosted/0.4.0/install/install-on-gcp-script.md:9]: I propose the following flow to more clearly communicate the difference between "just click this button" and "go through all these steps manually":

## Install Gitpod using Gitpod.io


The easiest way to install Gitpod on GCPis by using Gitpod.io and our automated installation script: `<button>`
This script...
Once....

## Or: Install manually
Alternatively, you can checkout the repository
 > repo

to your local machine and do the following steps manually. See the [manual GCP setup](../install-on-gcp-manual/) section for more details.

**Comment by geropl** [comment]
Somehow the link [you posted](https://github.com/gitpod-io/website/pull/620#issuecomment-624684209) does not match with the contents of this PR... :thinking:

File: src/docs/self-hosted/0.4.0/install/install-on-gcp-script.md (hunk 3 above)
Comment (jankeromnes) [reply src/docs/self-hosted/0.4.0/install/install-on-gcp-script.md:9]: Thanks again for the review! I agree with your suggestion, but I don't currently see how to implement it properly, so I'll leave it as a follow-up for later.

**Pull request merged**